utils/MyCurator.py

```diff
@@ -271,173 +398,144 @@ def process_generic_roi(self, roi):
             x_end,
             y_end,
             user_origin,
+            cached_stats
         )
 
     def curate_session(self, session: flywheel.Session):
 
         log.info(f"curating session {session.label}")
 
-        output_dict = {
-            "Group": [],
-            "Project": [],
-            "Subject": [],
-            "Session": [],
-            "Acquisition": [],
-            "File Name": [],
-            "File Type": [],
-            "Label": [],
-            "Description": [],
-            "Xmin": [],
-            "Xmax": [],
-            "Ymin": [],
-            "Ymax": [],
-            "User": [],
-            "ROI Type": [],
-        }
-        session=session.reload()
+        output_dict = copy.deepcopy(OUTPUT_TEMPLATE)
+        session = session.reload()
         session_info = session.info
 
         for pk in POSSIBLE_KEYS:
```

Comment:
Why loop over keys here if there is only one key for sessions?  Couldn't you just do `if 'ohifViewer' in session_info`?

Reply:
Yup!  Fixed